ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -33,21 +47,46 @@
         - kube-scheduler
         - kube-addons
         - etcd
+        - etcd2
       failed_when: false
 
     - name: Remove packages
       action: "{{ ansible_pkg_mgr }} name={{ item }} state=absent"
-      when: not is_atomic | bool
+      when: not is_coreos and not is_atomic | bool
       with_items:
         - flannel
         - kubernetes-node
         - kubernetes-master
         - etcd
+        - etcd2
```

Comment:
I think it might throw an error for non-coreos deployments. If it doesn't then it's no biggie. I'm submitting a follow-on PR that makes a few changes b/c I had an issue with this one. I'll cc you on the PR so you can review and test.